ui/src/api/application.ts

```diff
@@ -429,9 +429,10 @@ const getPlatformConfig: (application_id: string, type: string) => Promise<Resul
 const updatePlatformConfig: (
   application_id: string,
   type: string,
-  data: any
-) => Promise<Result<any>> = (application_id, type, data) => {
-  return post(`/platform/${application_id}/${type}`, data)
+  data: any,
+  loading?: Ref<boolean>
+) => Promise<Result<any>> = (application_id, type, data, loading) => {
+  return post(`/platform/${application_id}/${type}`, data, undefined, loading)
 }
 /**
  * 更新平台状态
```

Comment:
The provided function `updatePlatformConfig` has a couple of issues:

1. **Missing Parameters**:
   - The signature for `updatePlatformConfig` does not accept the `data` parameter when it is followed by an optional `loading` parameter.

2. **Incorrect Argument Passing**:
   - In the call to the `post` method within the implementation, there is no explicit value passed for the fourth argument (`headers`). It should be either undefined or explicitly set if needed.

Here's how you can fix these issues:

```typescript
const updatePlatformConfig: (
  application_id: string,
  type: string,
  data: any,
  loading?: Ref<boolean>
) => Promise<Result<any>> = (application_id, type, data, loading?) => {
  // Ensure that headers remain consistent with the post request
  return post(`/platform/${application_id}/${type}`, data, undefined, loading);
}
```

By adding the missing parameters and ensuring that all necessary arguments are correctly passed, the code will now work as expected without errors. This modification maintains clarity and correctness in the function definition and usage.